amazeeio-solutions/silverback-template
# SLB-188: Decap integration

## Description of changes

Integrate the Decap installation from #93 into Gatsby.

## Motivation and context

Provide a canonical way to use Decap in Silverback.

## How to test

1. Start a Gitpod with this PR
2. Go to `http://localhost:8000/admin`
3. CRUD some content
4. See the changes in `/apps/decap/data`
5. Rebuild the website to see pages in the frontend

> Will not work with "VSCode in Browser". Decap has to run on `localhost`.

## Review notes

I marked several occasions for improvements. Jira tickets for these have already been created.

## How has this been tested?
